app/src/main/java/com/keylesspalace/tusky/usecase/LogoutUsecase.kt

```diff
@@ -42,12 +43,13 @@ class LogoutUsecase @Inject constructor(
         disableUnifiedPushNotificationsForAccount(context, account)
 
         // disable pull notifications
-        if (!NotificationHelper.areNotificationsEnabled(context, accountManager)) {
-            NotificationHelper.disablePullNotifications(context)
+        if (!notificationService.areNotificationsEnabled()) {
+            // TODO this is working very wrong
```

Comment:
That would then be a next step.

Reply:
Could you give some more detials what the problem is?

Reply:
Mostly with the "thinking" behind it (or lack thereof). Earlier I had "worded very wrong" there.
It's "if notifications are disabled then disable them"?